src/components/chart-preview/ChartPreview.tsx

```diff
@@ -52,6 +55,42 @@ export const ActualChart = ({
     useSaveCsvData(csv, "chart-data");
   };
 
+  const onFigureDownloadClick = (e: any) => {
+    //TODO rework getting parent node
+    const fullFigureNode =
+      e.target.parentNode.parentNode.parentNode.parentNode.parentNode;
+
+    let figureCloneNode = fullFigureNode.cloneNode(true);
+    let emptyNode = document.createElement("div");
+    emptyNode.setAttribute(
+      "style",
+      "position: absolute; opacity: 0; pointerEvents: none; width: 1280px;",
+    );
+    figureCloneNode.setAttribute("style", "background-color: white;");
```

Comment:
Why do we have to set some styles on the emptyNode and some on the figureCloneNode when they end up appended together? Couldn't we do the appends first and all the styles in one go after?